### PR TITLE
Baby translations

### PR DESCRIPTION
**Pull request by crptm**
We can start with something like that. Honestly cant think of a good way to do it which doesnt involve babel transforms :D

### CODE REVIEW

File: common/actions/config.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+};
 
-export const CHANGE_LANGUAGE = (value: any) => Object({
-    type: CONFIG_LANGUAGE_CHANGE, value
-})
-
-export const CHANGE_NODE = (value: any) => Object({
-    type: CONFIG_NODE_CHANGE, value
-})
+export const CHANGE_NODE = (value: any) =>
```
Comment (dternyak) [inline common/actions/config.js]: Just whitespace changes? Already a huge PR, would be nice to avoid this.

Comment (crptm) [inline common/actions/config.js]: @dternyak yeah i am trying to not change files you could be touching, this is a prettier thing. One time formatting war :D

Comment (dternyak) [inline common/actions/config.js]: 👍

**Comment by crptm** [comment]
Merging

**Pull request merged**